platform/ios/src/MLNMapView+Metal.h

```diff
@@ -50,6 +50,9 @@ class MLNMapViewMetalImpl final : public MLNMapViewImpl,
     MLNBackendResource getObject() override;
     // End implementation of MLNMapViewImpl
 
+    virtual void setSynchronous(bool value) override { synchronousFrame = value; };
```

Comment:
nit: `virtual` isn't required when the `override` keyword is present, high compiler warning levels can complain if they follow the core guidelines on [virtual, override and final](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final)